agent/engine/common_integ_test.go

```diff
@@ -117,6 +117,17 @@ func verifyContainerRunningStateChangeWithRuntimeID(t *testing.T, taskEngine Tas
 		"Expected container runtimeID should not empty")
 }
 
+func verifyExecAgentRunningStateChange(t *testing.T, taskEngine TaskEngine) {
+	stateChangeEvents := taskEngine.StateChangeEvents()
+	event := <-stateChangeEvents
+	containerEvent := event.(api.ContainerStateChange)
+	assert.NotEmpty(t, containerEvent.ManagedAgents, "Expeceted exec-agent event has no managed agents")
```

Comment:
typo: Expected

Reply:
I think the error msg could be either "exec-agent event has no managed agents" or "Expected exec-agent event should have managed agents"

Reply:
fixed